docs/en/01-how-to-build/android.md

```diff
@@ -91,6 +91,12 @@ make install
 </code></pre>
    </td>
   </tr>
+  <tr>
+  <td>OpenJDK </td>
+  <td>It is necessary for building Java API.</br>
+  See <a href='../../../csrc/mmdeploy/apis/java/README.md'> Java API build </a> for building tutorials.
```

Comment:
This link is not available in readthedocs. May use the github URL of `README.md` instead.